README.md

```diff
@@ -1,2 +1,24 @@
-# hello-rails-react
-Testing rails with react in one app
+# README
```

Comment:
- [ ] Please make sure to update your README file with the necessary data to make it professional and descriptive. You can refer to this [template](https://github.com/microverseinc/readme-template).